test_s3.py

```diff
@@ -1,29 +1,45 @@
 import boto3
+import sys
+import logging
+
+# Create logger
+logger = logging.getLogger("MyLog-S3")
+logger.setLevel(logging.DEBUG)
+# create the logging handlers (file and stream)
+fh = logging.FileHandler("test_s3.log", mode='w')
+ch = logging.StreamHandler(stream=sys.stdout)
+fh.setLevel(logging.DEBUG)
+ch.setLevel(logging.INFO)
+# create formatter and add it to the handlers
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+fh.setFormatter(formatter)
+ch.setFormatter(formatter)
+# add the handlers to the logger
+logger.addHandler(ch)
+logger.addHandler(fh)
 
 # Create S3
 s3 = boto3.client('s3')
-
 bucket_name = 'test-bucket-boto-mzol'
```

Comment:
extract each s3 interaction to separate function

Reply:
done